SlaveWAMPServer.js

```diff
@@ -50,6 +50,10 @@ class SlaveWAMPServer extends WAMPServer {
 		});
 	}
 
+	/**
+	 * @param {object} request
+	 * @returns {{}}
```

Comment:
right, i will change `@param {object}` also to `@param {Object}` to be constistent with lisk-core https://github.com/LiskHQ/lisk/issues/691